webapp/components/test-file-results-table.js

```diff
@@ -399,7 +400,14 @@ class TestFileResultsTable extends WPTFlags(Pluralizer(AmendMetadataMixin(WPTCol
   }
 
   clearSelectedCells(selectedMetadata) {
-    this.handleClear(selectedMetadata, this.$['selected-toast']);
+    this.handleClear(selectedMetadata);
+  }
+
+  handleTriageMode(isTriageMode) {
+    if (isTriageMode && !this.pathIsATestFile) {
```

Comment:
Yea it is a tricky part. As a user toggles the triage button, `isTriageMode`, e.g. from False to True, is an observable change and invokes the `handleTriageMode` in `test-file-results-table.js` as well as `wpt-results.js`. So without this `pathIsATestFile` check, the paper toast from test-file-results-table.js and `wpt-results.js` will both pop up and lay on top of each other.

Reply:
One idea is to only use the paper toast in `wpt-results.js` but it requires data binding the properties needed for `wpt-amend-metadata` between `test-file-results-table.js` and `wpt-results.js` both way, which I am not a big fan.

Reply:
Looking at https://github.com/web-platform-tests/wpt.fyi/blob/02189b21fe72704be6ce11df17d3a6b246d2ad07/webapp/views/wpt-results.js#L187, this component is hidden behind `<dom-if>` so it shouldn't be active simultaneously with the one in `wpt-results`. Also I tried setting a breakpoint here and it does not trigger when I'm in `wpt-results`.

Reply:
What you said is correct, however if you set a breakpoint when you are in `test-file-results-table.js`, you will see the trigger. `isTriageMode` is passed from `wpt-results.js` to `test-file-results-table.js`. 

As another alternative, I can add ` <template is="dom-if" if="{{ pathIsATestFile }}">` to the paper-toast in wpt-results and remove the `!this.pathIsATestFile` check

Reply:
> What you said is correct, however if you set a breakpoint when you are in test-file-results-table.js, you will see the trigger. isTriageMode is passed from wpt-results.js to test-file-results-table.js, thus the trigger.

But in that case `pathIsATestFile` is already true? I think you only need to guard the toast in `wpt-results.js` to prevent it from showing up in the file view (either your current implementation or using a `dom-if` is fine), but IIUC there's no need to have that guard in `test-file-results-table.js` as the whole component isn't active when we are not in a file view.

Reply:
Yea that's right. I will remove the check in test-file-results-table.js